modules/manage/pages/manage-security/configure-ldap.adoc

```diff
@@ -135,33 +141,107 @@ This expands the dialog vertically, as follows:
 [#configure-ldap-dialog-enable-ldap-user-auth-field]
 image::manage-security/configureLdapEnableLdapUserAuthField.png[,400,align=left]
 +
-This provides three options whereby usernames can be mapped on the LDAP server.
+This provides three options whereby usernames can be mapped to LDAP distinguished names (https://ldap.com/ldap-dns-and-rdns/[DN^]).
 The default option is *Template*.
-An appropriate template should be entered into the *Template* editable text field.
+An appropriate mapping should be entered into the *Template* editable text field: this avoids the need to request the LDAP server to provide the username-to-DN mapping.
+The required format is indicated by the placeholder text, within the field.
```

Comment:
Format is described in RFC 4514 (https://tools.ietf.org/html/rfc4514).

Reply:
I have added the link.